astropy/time/core.py

```diff
@@ -112,6 +113,23 @@
         'IAU1994': {'function': erfa_time.gst94, 'scales': ('ut1',)}}}
 
 
+class TimeInfo(BaseInfo):
+    """
+    Make dtype and unit be None and be read-only.  Setting attrs_from_parent
+    to these read-only values is needed so they don't get copied.
+    """
+    attrs_from_parent = set(['unit'])
+
+    @property
+    def unit(self):
+        return None
+
+    info_summary_stats = staticmethod(
+        data_info_factory(names=BaseInfo._stats,
```

Comment:
Yes, that was what I worried about. But of course, it is captured in the try/except in the function produced by `data_info_factory`... So this doesn't matter, sorry.